src/SFA.DAS.ApplyService.Web/Startup.cs

```diff
@@ -157,9 +160,12 @@ private void ConfigHttpClients(IServiceCollection services)
             })
             .SetHandlerLifetime(handlerLifeTime);
 
-            services.AddHttpClient<ICharityCommissionApiClient, CharityCommissionApiClient>(config =>
+            services.AddHttpClient<IOuterApiClient, OuterApiClient>(config =>
             {
-                config.BaseAddress = new Uri(_configService.InternalApi.ApiBaseAddress);
+                config.BaseAddress = new Uri(_configService.OuterApiConfiguration.ApiBaseUrl);
+                config.DefaultRequestHeaders.Add(acceptHeaderName, acceptHeaderValue);
```

Comment:
Avoid variables, use literal strings to be consistent with other headers.

Reply:
Done